content/en/docs/contribute/new-content/contributing-content.md

```diff
@@ -0,0 +1,503 @@
+
+    ```bash
+    git fetch origin
+    git fetch upstream
+    ```
+
+    This makes sure your local repository is up to date before you start making changes.
+
+    {{< note >}}
+    This workflow is different than the [Kubernetes Community GitHub Workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md). You do not need to merge your local copy of `master` with `upstream/master` before pushing updates to your fork.
+    {{< /note >}}
+
+### Create a branch
+
+1. Decide which branch base to your work on:
+
+  - For improvements to existing content, use `upstream/master`.
+  - For new content about existing features, use `upstream/master`.
+  - For localized content, use the localization's conventions. For more information, see [localizing Kubernetes documentation](/docs/contribute/localization/).
+  - For new features in an upcoming Kubernetes release, use the feature branch. For more information, see [documenting for a release](/docs/contribute/new-content/new-features/).
+  - For long-running efforts that multiple SIG Docs contributors collaborate on,
+    like content reorganization, use a specific feature branch created for that
+    effort.
+
+    If you need help choosing a branch, ask in the `#sig-docs` Slack channel.
+
+2. Create a new branch based on the branch identified in step 1. This example assumes the base branch is `upstream/master`:
+
+    ```bash
+    git checkout -b <my_new_branch> upstream/master
+    ```
+
+3.  Make your changes using a text editor.
+
+At any time, use the `git status` command to see what files you've changed.
+
+### Commit your changes
+
+When you are ready to submit a pull request, commit your changes.
+
+1. In your local repository, check which files you need to commit:
+
+    ```bash
+    git status
+    ```
+
+    Output is similar to:
+
+    ```bash
+    On branch <my_new_branch>
+    Your branch is up to date with 'origin/<my_new_branch>'.
+
+    Changes not staged for commit:
+    (use "git add <file>..." to update what will be committed)
+    (use "git checkout -- <file>..." to discard changes in working directory)
+
+    modified:   content/en/docs/contribute/new-content/contributing-content.md
+
+    no changes added to commit (use "git add" and/or "git commit -a")
+    ```
+
+2. Add the files listed under **Changes not staged for commit** to the commit:
+
+    ```bash
+    git add <your_file_name>
+    ```
+
+    Repeat this for each file.
+
+3.  After adding all the files, create a commit:
+
+    ```bash
+    git commit -m "Your commit message"
+    ```
+
+    {{< note >}}
+    Do not reference a GitHub issue or pull request by ID or URL in the
+    commit message. You can link issues and pull requests together later in the GitHub UI.
+    {{< /note >}}
+
+4. Push your local branch and its new commit to your remote fork:
+
+    ```bash
+    git push origin <my_new_branch>
+    ```
+
+Next:
```

Comment:
nit: Since the rest of the Contribute sections where streamlined, these new content additions (fork+clone, rebasing, squashing) could be pared down.
nit: The `Next` and the two items after don't seem to fit in when you have sub-headings immediated after.